bitmark-references/bitmark-node-software/bitmark-shares.md

```diff
@@ -0,0 +1,70 @@
+# Bitmark shares
+
+## Reasoning
+
+* There may be some reason to turn a property into shares in order to
+  divide the ownership.
+
+* Once a property has been divided it does not make sense to combine
+  it back into a single unit again.
+
+* The number of shares initially created is a fixed amount and cannot
+  be changed as this would fundamentally alter the valuation of one
+  share.
+
+* Shares can be freely traded in any quantity without keeping a linked
+  history.  i.e. there is no real order to share transactions after
+  initial creation.
+
+## Actions that can be performed
+
+* Shares can be created.
+
+* Shares can be granted from the original supply to others.
+
+* Accounts holding shares can effectively trade them, but only among
+  holders of a single Bitmark Share.  This leads to a requirement of a
```

Comment:
It should be among holders of the same type of share (same share ID)?

Reply:
yes, will add this